lib/v2/components/snack_bar_info.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class SnackBarInfo extends SnackBar {
```

Comment:
@gguij004 
@RaulUrtecho 
@n13 
@fbarzin 

What do u guys think if every time we add a new `global` component, we post in the discord dev channel letting others know about it? 

Perhaps even add a screen shot. 

This would make it easier to know whats available already.

Reply:
Sounds like a good idea